### ISSUE TITLE
Publish on Flathub

### ISSUE THREAD

**Issue by marek22k** [tobspr-games/shapez.io#1573]
Flatpak is a package management software that offers to install software across distributions with a built-in sandbox. Flathub is the standard repository for Flatpak. I would like to see shapez published on Flathub.

**Comment by WaffleDevs** [comment]
Just install steam

**Comment by EmeraldBlock** [comment]
for more context, the game's sales model is basically "you can buy a build (the game on Steam) or build it yourself from source", so offering a free official alternative kind of defeats the purpose.

as for selling on other platforms in general, the itch.io release was abandoned a while ago (and it seems to have been taken down entirely?)

**Comment by marek22k** [comment]
> Just install steam

Steam ist not FLOSS, it is propritary.

**Comment by Xanderplayz16** [comment]
> > Just install steam
> 
> Steam ist not FLOSS, it is propritary.

Yeah who cared if you really hate Steam go compile the game yourself (It is actually relatively difficult if you don't do everything right)
